docs-ref-mapping/reference-latest.yml

```diff
@@ -703,79 +703,6 @@
       landingPageType: Service
       children:
       - com.microsoft.azure.management.streamanalytics*
-  - name: Spring Boot
-    uid: azure.java.sdk.landingpage.services.springboot
-    href: ~/docs-ref-services/latest/spring-boot-readme.md
-    landingPageType: Service
-    items:
-    - name: Spring Boot Starter
-      uid: azure.java.sdk.landingpage.services.springboot.starter
-      href: ~/docs-ref-services/latest/spring-boot-starter-readme.md
```

Comment:
I want to delete these md files, but I'm not sure whether there is some job will sync these files from azure-sdk-for-java repo again.

Reply:
Hey @chenrujun , We're pulling the entire `spring boot` hierarchy. I'm ok with that, if they exist in another location. What I want to follow-up on is if I can remove the generated content as well.

Reply:
Hi, @scbedd .

Thank you very much for your information.

>  if I can remove the generated content as well.

Yes, please. It's better to remove the generated contents.

Reply:
Awesome! I'll take care of this and rip any onboarded spring project out.